workspaces/quickstart/app-config.yaml

```
@@ -3,62 +3,86 @@ app:
   baseUrl: http://localhost:3000
   quickstart:
     - title: Set up authentication
+      titleKey: steps.title.setupAuthentication
       icon: Admin
       roles:
         - admin
       description: Set up secure login credentials to protect your account from unauthorized access.
+      descriptionKey: steps.description.setupAuthentication
       cta:
         text: Learn more
+        textKey: steps.cta.learnMore
```

Here I would also mention in the key hat this is a title (or a buttonTitle)

```suggestion
        textKey: steps.learnMore.ctaTitle
```